pycbc/inference/models/relbin.py

```diff
@@ -679,8 +682,9 @@ def _loglr(self):
         lik = self.likelihood_function
         norm = 0.0
         filt = 0j
+        pol_phase = numpy.exp(-2.0j * p['polarization'])
 
-        self.snr_draw(self.get_snr(wfs))
+        self.snr_draw(wfs)
```

Comment:
This doesn't look right. `snr_draw` should take snrs as input, not waveforms?!

Reply:
Ah I see, you changed the function to use key-word arguments, but then didn't use it here because wfs comes first. Don't like that at all! If this is a key-word argument then give it as a key-word argument so that this is clear.

Reply:
Good point, I've changed it to be explicit